.github/workflows/webpack.yml

```diff
@@ -0,0 +1,31 @@
+name: Build Metronome
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout 🛎️
+      uses: actions/checkout@master
+      
+    - name: Setup Node 🔧
+      uses: actions/setup-node@v3
+      with:
+        node-version: '14'
+      
+    - name: Install and Build 🔧
+      run: |
+        yarn install
+        yarn build
+
+    - name: Deploy 🚀
+      uses: JamesIves/github-pages-deploy-action@v4.2.5
+      with:
```

Comment:
i see! let's see if this does the trick. if we need to, we can always switch to `gh-branches` if this doesn't work

Reply:
Okie!